srcpkgs/splatmoji/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'splatmoji'
+pkgname=splatmoji
+version=1.2.0
+revision=1
+conf_files="/etc/xdg/${pkgname}/${pkgname}.config"
+hostmakedepends="pandoc"
+depends="bash>4.4 rofi xdotool xsel jq"
```

Comment:
drop the `>4.4`

Reply:
I just took it from the package documentation